template/src/routes/stories/app-link.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import {
+  AppLink, RouteName,
+  Router, routes,
+} from '../index';
+
+export default {
+  title: 'AppLink',
+  component: AppLink,
+  decorators: [
+    (story) => {
+      const routesMapped = routes.map((route) => ({
+        ...route,
+        component: story,
+      }));
+
+      return (
+        <Router routes={routesMapped} />
+      );
+    },
+  ],
+} as ComponentMeta<typeof AppLink>;
+
+export const Default: ComponentStory<typeof AppLink> = ({
+  // eslint-disable-next-line react/prop-types
```

Comment:
I think this rule can be disabled project-wide. What do you think? When we introduced TS we did not disable this one.